src/controllers/article_tag.js

```diff
@@ -0,0 +1,26 @@
+import { Article } from '../db_api/models/index';
```

Comment:
@toluola Naming convention of the file is not conventional with the team. Kindly review

Reply:
fixed